Utils/request_contributor_review.py

```diff
@@ -15,6 +26,15 @@
 XSOAR_SUPPORT = "xsoar"
 PACK_METADATA_GITHUB_USER_FIELD = "githubUser"
 PR_COMMENT_PREFIX = "pack has been modified on files:\n"
+PACK_METADATA_SUPPORT_EMAIL_FIELD = "email"
+PACK_METADATA_DEV_EMAIL_FIELD = "developerEmail"
+GMAIL_CLIENT_ID = "391797357217-pa6jda1554dbmlt3hbji2bivphl0j616.apps.googleusercontent.com"
```

Comment:
This should also be a secret

Reply:
It was global variable at the GmailSingleUser application too...